batch7crud-roll1/src/main/webapp/resources/js/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Created by kiran on 2/1/16.
+ */
+var deleteButton = document.getElementsByClassName("delete");
+for (var i = 0; i < deleteButton.length; i++) {
+    deleteButton[i].onclick = function (e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define this function outside of a loop. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3AFunctionDefinitionInsideLoop)